src/utils/useScrollIntoView.js

```diff
@@ -6,7 +6,7 @@ export const useScrollIntoView = (deps) => {
     if (ref.current) {
       ref.current.scrollIntoView({ behavior: "smooth" });
     }
-  }, deps);
+  }, [deps]);
```

Comment:
this shouldn't be wrapped in an array if this is to fix the lint warning we can just use a comment to ignore

Reply:
ok I'll throw in that ignore